class CustomExceptions(Exception):
    pass
class FileOpenError(CustomExceptions):
    pass
